Traceback (most recent call last):
  File "main.py", line 2, in <module>
    import uvicorn
ModuleNotFoundError: No module named 'uvicorn'
Traceback (most recent call last):
  File "main.py", line 2, in <module>
    import uvicorn
ModuleNotFoundError: No module named 'uvicorn'
